.travis.yml

```diff
@@ -5,14 +5,15 @@ sudo: false
 python:
   - 3.7
   - 3.8
+  - 3.9
 
 env:
   - DJANGO=2.2
   - DJANGO=3.0
   - DJANGO=3.1
 
 addons:
-  postgresql: "9.6"
+  postgresql: "10"
```

Comment:
Stick with 9.6, till we upgrade production

Reply:
ok!